gtsam/gtsam.i

```diff
@@ -1809,7 +1809,7 @@ class GaussianFactorGraph {
 };
 
 #include <gtsam/linear/GaussianConditional.h>
-virtual class GaussianConditional : gtsam::GaussianFactor {
+virtual class GaussianConditional : gtsam::JacobianFactor {
```

Comment:
Why change the super class for this? Doesn't GaussianFactor make sense for a GaussianConditional?

Reply:
JacobianFactor is in turn a GaussianFactor